NEWS.md

```diff
@@ -1,4 +1,13 @@
-## 2.10.0 - Unreleased
+## 3.1.0 - Unreleased
+
+## 3.0.0 - Released (Quesnelia R1 2024)
+The primary focus of this release was to update vertx and edge-common versions
+
+[Full Changelog](https://github.com/folio-org/edge-orders/compare/v2.9.0...v3.0.0)
```

Comment:
there is stories to add
https://folio-org.atlassian.net/projects/EDGORDERS/versions/11580/tab/release-report-all-issues

Reply:
and also fix versions should be changed of stories

Reply:
these 2 stories are about the same, as for me it's ok to include only single of them

Reply:
These stories are module version updates so according to guidelines they are included in `Dependencies` section but not in `Stories` section as there are no code changes.

Reply:
then, let's include at least one of them in stories section to show some effort made in this module